apps/system/style/action_menu/action_menu_extended.css

```diff
@@ -136,3 +136,9 @@ form[role="dialog"][data-type="object"].visible {
     z-index: -1;
   }
 }
+
+/* RTL View */
+html[dir="rtl"] form[role="dialog"][data-type="action"] > menu > button.icon,
+html[dir="rtl"] form[role="dialog"][data-type="object"] > menu > button.icon {
+  background-position: right 0.5rem center;
+}
```

Comment:
I think we better leave it there for consistency with other files that contain RTL definitions, we've always putted them at the bottom of the file to make finding them easier.
What do you think?